### PR TITLE
show symbol hovered over

### PR DESCRIPTION
**Pull request by zZeck**

### CODE REVIEW

File: Tests/VsCodeExtension/Requests.fs [hunk 1]
```diff
@@ -489,11 +489,9 @@ let initialize(solutionPath: string) =
     }}
 }}"
 
-let initialized() = @"{""jsonrpc"":""2.0"",""method"":""initialized"",""params"":{},""id"":10}}"
+let textDocumentDocumentSymbol = @"{ ""jsonrpc"":""2.0"", ""method"": ""textDocument/documentSymbol"", ""id"":1 }"
 
-let textDocumentDocumentSymbol() = @"{""jsonrpc"":""2.0"",""method"":""textDocument/documentSymbol"",""id"":1}"
```
Comment (TrevorThoele2) [inline Tests/VsCodeExtension/Requests.fs:494]: Can we reinstate master's version of `textDocumentDocumentSymbol` here? Seems like you didn't want to change that.

File: Tests/VsCodeExtension/Requests.fs [hunk 2]
```diff
@@ -489,11 +489,9 @@ let initialize(solutionPath: string) =
     }}
 }}"
 
-let initialized() = @"{""jsonrpc"":""2.0"",""method"":""initialized"",""params"":{},""id"":10}}"
```
Comment (TrevorThoele2) [inline Tests/VsCodeExtension/Requests.fs:492]: I think we should reinstate master's version of this as well.

File: Tests/VsCodeExtension/ServerTests.fs [hunk 3]
```diff
@@ -94,22 +94,18 @@ let ``server responds with null when requesting initialized`` () = async {
...
 
     inputWriter.Write(prependContentLength(Requests.initialize(
-        Path.Join(Directory.GetCurrentDirectory(), "Solutions", "Basic", "Basic.sln"))))
+        Path.Join(Directory.GetCurrentDirectory(), "Solutions", "Basic"))))
     let initializeOutput = readOutput(outputReader)
     initializeOutput.Should().NotBeNull("", []) |> ignore
 
-    inputWriter.Write(prependContentLength(Requests.initialized()))
-    let initializedOutput = readOutput(outputReader)
-    initializedOutput.Should().Be(@"{""jsonrpc"":""2.0"",""id"":10,""result"":null}", "", []) |> ignore
```
Comment (TrevorThoele2) [inline Tests/VsCodeExtension/ServerTests.fs:103]: If you remove lines 101-103, I don't think this test makes any sense anymore. Can you reinstate these lines as well?

File: VsCodeExtension/Server/Core.fs [hunk 4]
```diff
@@ -126,7 +129,9 @@ let SemanticTokenModifiers =
 let initialize(input: Input<InitializeParams>): Async<Output<InitializeResult>> = async {
     if (not MSBuildLocator.IsRegistered) then MSBuildLocator.RegisterDefaults() |> ignore
     let workspace = MSBuildWorkspace.Create()
-    let! _ = (workspace.OpenSolutionAsync(input.Parameters.RootPath.Value) |> Async.AwaitTask)
+    let slnPaths = System.IO.Directory.GetFiles(input.Parameters.RootPath.Value, "*.sln")
```
Comment (TrevorThoele2) [inline VsCodeExtension/Server/Core.fs:132]: Can we name this `solutionPaths`? Makes sense to me to not use contractions in names.

File: VsCodeExtension/Server/Core.fs [hunk 5]
```diff
@@ -126,7 +129,9 @@ let SemanticTokenModifiers =
...
+    let sln = slnPaths |> Seq.head
+    let! _ = (workspace.OpenSolutionAsync(sln) |> Async.AwaitTask)
```
Comment (TrevorThoele2) [inline VsCodeExtension/Server/Core.fs:134]: Can this just be `do! workspace.OpenSolutionAsync(sln) |> Async.AwaitTask`?

Comment (zZeck) [reply VsCodeExtension/Server/Core.fs:134]: I get an error about
`This expression was expected to have type
    'Solution'    
but here has type
    'unit' `

Comment (TrevorThoele2) [reply VsCodeExtension/Server/Core.fs]: Maybe it'd have to be `do! (workspace.OpenSolutionAsync(sln) |> Async.AwaitTask)`. Probably not worth anymore consideration right now.

**Review by TrevorThoele2 — APPROVED** [review]
Thanks for looking at that.

**Pull request merged**